packages/security/gnutls/package.mk

```diff
@@ -3,11 +3,11 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="gnutls"
-PKG_VERSION="3.6.5"
-PKG_SHA256="073eced3acef49a3883e69ffd5f0f0b5f46e2760ad86eddc6c0866df4e7abb35"
+PKG_VERSION="3.6.10"
+PKG_SHA256="b1f3ca67673b05b746a961acf2243eaae0ffe658b6a6494265c648e7c7812293"
 PKG_LICENSE="LGPL2.1"
 PKG_SITE="https://gnutls.org"
-PKG_URL="https://www.gnupg.org/ftp/gcrypt/gnutls/v${PKG_VERSION%.*}/gnutls-$PKG_VERSION.tar.xz"
+PKG_URL="https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-$PKG_VERSION.tar.xz"
```

Comment:
is `v${PKG_VERSION%.*}` not working anymore ?

Reply:
gnutls occasionally uses a 4th tuple (ie 3.6.7.1) which that broke on.